### ISSUE TITLE
False positive for spacing around "=>" in inline array within foreach 

### ISSUE THREAD

**Issue by Pittiplatsch** [squizlabs/PHP_CodeSniffer#434]
```
foreach ([
        'foo'           => 'bar',
        'foobaz'        => 'bazzy',
    ] as $key => $value) {
}
```
"Expected 1 space before "=>"; 11 found"

**Comment by aik099** [comment]
I think it's not false positive because your code should look like this instead:

```php
foreach ([
        'foo'    => 'bar',
        'foobaz' => 'bazzy',
    ] as $key => $value) {
}
```

**Comment by Pittiplatsch** [comment]
You just removed some spaces before the "=>" arrows, right? This doesn't change anything. My example has so many spaces as I left out some items which have longer keys...

**Comment by aik099** [comment]
Ah, I thought that these extra leading spaces before `=>` where causing the issue.

**Comment by gsherwood** [comment]
I found incorrect values in the error message, but I can't see any false positives in code like this. You may need to paste actual failing code that is formatted correctly rather than just a snippet.

**Comment by gsherwood** [comment]
> I found incorrect values in the error message

Sorry, I was wrong. Error message was correct in my sample code.

I've replicated the error you are getting. I was looking into the wrong sniff.

**Comment by gsherwood** [comment]
This is now fixed.

**Comment by Pittiplatsch** [comment]
Thx :-)
